ml-agents/mlagents/trainers/components/reward_signals/gail/model.py

```diff
@@ -261,5 +292,8 @@ def create_loss(self, learning_rate: float) -> None:
             )
         else:
             self.loss = self.discriminator_loss
+
+        self.loss = self.loss + self.gradient_penalty * self.compute_gradient_penalty()
```

Comment:
I think we need better var names here. From the current names I would expect `self.compute_gradient_penalty()` to return the gradient penalty, but it returns the magnitude of the gradient.

Reply:
Changed this to `create_gradient_magnitude` and the weight to be `gradient_penalty_weight`